src/pages/Login/model/login.schema.ts

```diff
@@ -0,0 +1,7 @@
+import { string } from "yup"
+
+import { UserSchema } from "~/entities/user/model/user.schema"
```

Comment:
It should be imported like:
`import { UserSchema } from "~/entities"`

Reply:
Done